repo/packages/H/hue/0/config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "type": "object",
+  "properties": {
+    "hdfs": {
+      "description": "Hadoop specific configuration properties",
+      "type": "object",
+      "properties": {
+        "scheduler-uri": {
+          "description": "HDFS package artifact tarball (must begin with 'hdfs')",
+          "type": "string",
+          "default": "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz"
+        }
+      },
+      "required": [
+        "scheduler-uri"
+      ]
+    },
+    "hue": {
+      "description": "Hue specific configuration properties",
+      "type": "object",
+      "properties": {
+        "configuration": {
+          "description": "A custom universally-accessible URL of a hue.ini file.",
+          "type": "string",
```

Comment:
Let's make this "https://s3.amazonaws.com/downloads.mesosphere.io/hue/httpfs-site.xml" instead of "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/hue/httpfs-site.xml"